documentation/src/main/asciidoc/topics/con_xsite_state_transfer.adoc

```diff
@@ -0,0 +1,29 @@
+[id='xsite_state_transfer-{context}']
+= State Transfer
+State transfer is an administrative operation that synchronizes data between
+sites.
+
+For example, **LON** goes offline and **NYC** starts handling client requests.
+When you bring **LON** back online, the {brandname} cluster in **LON** does not
+have the same data as the cluster in **NYC**.
+
+To ensure the data is consistent between **LON** and **NYC**, you can push
+state from **NYC** to **LON**.
+
+* State transfer is bidirectional. For example, you can push state from **NYC**
+to **LON** or from **LON** to **NYC**.
+* Pushing state to offline sites brings them back online.
+* State transfer overwrites only data that exists on both sites, the originating site and the receiving site.
++
+For example, "k1" exists on **LON** and **NYC**. "k2" exists on **LON** only.
+If you push state from **NYC** to **LON**, the transfer does not affect "k2" on
+**LON**. {brandname} does not delete or overwrite "k2".
```

Comment:
I guess you should say: brandname does not delete data.

There is another example that may useful. If k2 exists in LON and NYC, and k2 is removed in NYC while LON is offline, the state transfer doesn't delete it.

Also, it may be useful to suggest to clear the cache in LON before pushing the state from NYC => LON